modules/040-control-plane-manager/images/etcd/werf.inc.yaml

```diff
@@ -1,25 +1,30 @@
 ---
 image: {{ $.ModuleName }}/{{ $.ImageName }}
-from: {{ $.Images.BASE_ALPINE }}
+fromImage: common/distroless
 import:
   - image: common/pause
     add: /pause
     to: /pause
     before: setup
   - artifact: {{ $.ModuleName }}/{{ $.ImageName }}-artifact
-    add: /usr/local/bin
+    add: /
     to: /usr/bin
     includePaths:
     - etcd
     - etcdctl
     before: setup
-shell:
-  install:
-  - mkdir -p /var/etcd /var/lib/etcd
-  - |
-    echo "hosts: files mdns4_minimal [NOTFOUND=return] dns mdns4" >> /etc/nsswitch.conf
 docker:
```

Comment:
Why we remove this ?

Reply:
It's not required at all. Even etcd's upstream removed it.

https://github.com/etcd-io/etcd/pull/15155